core/platform/cache/redis_cache_services.py

```diff
@@ -22,10 +22,22 @@
 from core.domain import caching_domain
 import feconf
 import python_utils
+
 import redis
 
-REDIS_CLIENT = redis.Redis(
-    host=feconf.REDISHOST, port=feconf.REDISPORT)
+# Redis client for our own implementation of caching.
+OPPIA_REDIS_CLIENT = redis.StrictRedis(
+    host=feconf.REDISHOST,
+    port=feconf.REDISPORT,
+    db=feconf.OPPIA_REDIS_DB_NUMBER
```

Comment:
Hmm, I was a bit vary to use index since it has its own meaning in databases. google searches for both "number" and "index" are quite similar and lead to explanations that explain the same thing. But I agree that index is probably a bit more official in the Redis context.